client/sql/sparksession.go

```diff
@@ -154,3 +163,17 @@ func (s *sparkSessionImpl) analyzePlan(plan *proto.Plan) (*proto.AnalyzePlanResp
 	}
 	return response, nil
 }
+
+func consumeExecutePlanClient(responseClient proto.SparkConnectService_ExecutePlanClient) error {
```

Comment:
Can you add a bit doc on how the data is consumed and how it deals with streaming results etc.

Reply:
Yes, added.